usr/src/test/zfs-tests/tests/functional/rsend/send-cD.ksh

```diff
@@ -46,7 +46,7 @@ log_must zfs create -o compress=lz4 $sendfs
 log_must zfs create -o compress=lz4 $recvfs
 typeset dir=$(get_prop mountpoint $sendfs)
 # Don't use write_compressible: we want compressible but undedupable data here.
-log_must cp /kernel/genunix $dir/file
+log_must cp /kernel/amd64/genunix $dir/file
```

Comment:
will be better to use some define a variable ${MACH64}, because amd64 available only on Intel, but on sparc we have sparcv9
maybe for this test we can use another binary file and to be a more universal?

Reply:
Makes sense.